### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by simonjiao**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

File: commons/service-registry/src/service_registry.rs [hunk 1]
```diff
@@ -492,14 +492,14 @@ where
     }
 }
 
-pub struct PutShardRequest<T>
```
Comment (nkysg) [inline commons/service-registry/src/service_registry.rs:495]: 这个文件为啥有修改？

Comment (simonjiao) [reply commons/service-registry/src/service_registry.rs:495]: 这个是typo。get_shared/put_shared会操作对应的Request.我的理解是SharedRequest，不是ShardRequest。

**Review by sanlee42 — APPROVED** [review]

**Comment by sanlee42** [comment]
runing cargo fmt && cargo clippy for the code formating and checking

**Comment by nkysg** [comment]
bash  ./scripts/check_commit.sh

**Pull request merged**